examples/github-pages/README.md

```diff
@@ -19,7 +19,10 @@ pnpm create next-app --example github-pages nextjs-github-pages
 ### Deploy to GitHub Pages
 
 1.  Create a new public GitHub repository.
-1.  Edit `next.config.js` to match your GitHub repository name.
+1.  Edit `next.config.js` to match your GitHub repository name:
+    - Given the pattern `https://github.com/<user>/<repo>`, update the two fields: `basePath` and `assetPrefix`:
+      - `basePath: "/repo"`
+      - `assetPrefix: "/repo"`
```

Comment:
```suggestion
1.  Edit `next.config.js` to match your GitHub repository name:
    - Given the pattern `https://github.com/<user>/<repo>`, update your `basePath` config to `/repo`. 
```
`basePath` automatically sets `assetPrefix` as well so both don't need to be manually set.